src/ontology/cl-edit.owl

```diff
@@ -32333,6 +32334,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34497389") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4047006 "fetal artery endothelial cell")
 SubClassOf(obo:CL_4047006 obo:CL_1000413)
 
+# Class: obo:CL_4047007 (immature pericyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28564607") Annotation(oboInOwl:hasDbXref "PMID:32466671") Annotation(oboInOwl:hasDbXref "PMID:34497389") Annotation(oboInOwl:hasDbXref "doi:10.1016/j.devcel.2011.07.001") Annotation(oboInOwl:hasDbXref "doi:10.3389/fgene.2022.1056114") obo:IAO_0000115 obo:CL_4047007 "A pericyte that is in an early stage of development, found in newly forming or remodeling blood vessels. An immature pericyte is characterised by it's mesenchymal stem cell-like properties and high proliferative capacity, which allows it to differentiate into various types of pericytes and contribute to the structural and functional maturation of the vasculature. Immature pericytes are stellate in new vessels and elongated with less protrusions in remodling vessels.")
```

Comment:
Please, use PMID when they are available
doi:[10.1016/j.devcel.2011.07.001](https://doi.org/10.1016/j.devcel.2011.07.001)-> PMID:21839917
doi:[10.3389/fgene.2022.1056114](https://doi.org/10.3389/fgene.2022.1056114)-> PMID:36685855

Reply:
amended these to PMID